### ISSUE TITLE
[RFE] Additional parameters for custom error handler

### ISSUE THREAD

**Issue by FrostyX** [simpleapples/flask-wtf-decorators#2]
Let's say I have a custom error handler function implemented like this.

    @forms.validator.error_handler
    def error_handler(errors):
        for field_errors in errors.values():
            for error in field_errors:
                flask.flash(error, category="danger")
        return render_template("search_results.html")

I would really like to have access to the whole `form` object within this method because I would like to return something like

    return render_template("search_results.html", previous_search=form.package_name.data)

**Comment by simpleapples** [comment]
It make sense to add variable `form` to method `error_handler` as the second varaible. 
How do you feel about this?

**Comment by FrostyX** [comment]
I think so. The problem is, that by doing so, we will break existing solutions for people that have `def error_handler(errors):` and not `def error_handler(errors, *args, **kwargs):` because they will get an unexpected number of arguments to their function.

**Comment by simpleapples** [comment]
Yep, adding a new error handler method is a better way to be compatible with old code.
